File: Java Programming/Competitive Programming/Coding Challenges 10/program2.java
//Q2.Write a program which accept number from user and print even factors of that number.
//Input : 24
//Output: 1 2 4 6 8 12

import java.util.Scanner;

class program2
{
    static void DisplayFactor(int iNo)
    {
        int i = 0;

        if (iNo <= 0)
        {
            iNo = -iNo;
        }

        for (i = 1; i <= iNo / 2; i++)
        {
            if ((iNo % i == 0) && ((i % 2 == 0) || (i == 1)))
            {
                System.out.print(i + "\t");
            }
        }
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter number: ");
        iValue = sobj.nextInt();

        DisplayFactor(iValue);

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 9/program2.cpp
//Q2.program which accept number from user and print numbers till that number
//Input : 8
//Output : 1 2 3 4 5 6 7 8

#include <iostream>
using namespace std;

void Display(int iNo)
{
    if (iNo < 0)
    {
        iNo = -iNo;
    }

    int i = 1;

    cout << "Output is: ";

    while (i <= iNo)
    {
        cout << i << "\t";
        i++;
    }
}

int main()
{
    int iValue = 0;

    cout << "Enter Number: ";
    cin >> iValue;

    Display(iValue);

    return 0;
}